section4/reflection.md

```diff
@@ -2,21 +2,57 @@
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
 
+  - I wound up using a digital timer, but I plan to acquire a mechanical one to experience the full effect of the technique. Even with the digital timer I realized how much time I waste clicking though email, random google searches, etc.
```

Comment:
Good observation! I have to force myself to close all non-work tabs, hide my phone, etc. I put time on my calendar for things like "catch up on texts, social media" so that in my head I know I will have time to do it and am less tempted during essential work hours. This will be super important for you to find whatever works for you as you start full time, you won't have a minute to waste. Let me know if you'd like to brainstorm other strategies.